exercises/4-an-enslaved-hope/README.md

```diff
@@ -161,21 +161,21 @@ git push
 
 1. Change directory to `todolist`
 ```bash
-cd /projects/todolist
+cd todolist
```

Comment:
@xandradx I think we still want to keep this as `cd /projects/todolist` as this is the full path to where the `todolist` app livees